drivers/base/bootvid/CMakeLists.txt

```diff
@@ -35,6 +35,14 @@ add_library(bootvid MODULE
     bootvid.rc
     ${CMAKE_CURRENT_BINARY_DIR}/bootvid.def)
 
+if(SANITIZE_UB OR KASAN_ENABLED)
```

Comment:
That whole thing certainly needs to be under a `if(CMAKE_C_COMPILER_ID STREQUAL "Clang")`, because MSVC (at least) doesn't support that.